SwiftScripts/Sources/Codegen/Target.swift

```diff
@@ -4,23 +4,17 @@ import ArgumentParser
 
 enum Target {
     case starWars
-    case starWarsSwiftCodegen
     case gitHub
     case upload
-    case animalKingdom
     
     init?(name: String) {
         switch name {
         case "StarWars":
             self = .starWars
-        case "StarWars-SwiftCodegen":
-            self = .starWarsSwiftCodegen
         case "GitHub":
             self = .gitHub
         case "Upload":
             self = .upload
-        case "AnimalKingdom":
-            self = .animalKingdom
```

Comment:
did we mean to delete this?

Reply:
I did, it was only there to be used with `swiftExperimental`. It comes back in the other RFC-focused branch.